src/makeInfinityAdder.js

```diff
@@ -6,7 +6,69 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  const adder = (a) => {
+    if (!a && a !== 0) {
+      return 0;
+    }
+
+    return (b) => {
+      if (!b && b !== 0) {
```

Comment:
You should use `closure` and store result of the sum;

Function `adder` should return itself if we pass the param. If not - we should return the counter